packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.scss

```diff
@@ -60,10 +60,9 @@
       div,
       span,
       p {
-        font-size: 16px !important;
         line-height: 24px !important;
         /* identical to box height, or 143% */
-
+        font-family: 'Silka';
```

Comment:
Hmm, after those changes, it is even messier than it used to be before. [This design](https://www.figma.com/file/ArBB0hnZzAxvD3UCQF6AQj/Editor-improvements?type=design&node-id=10%3A6548&mode=design&t=Uye7QVJXxEssfCyZ-1) clearly states that text in quill editor, thread and comment should be the same for particular text type like body, h1, h2 etc. Right now, for body has three different sizes and fonts:
1. Thread - 14px, Neue Haas Unica
2. Quill editor - 13px, Helvetica
3. Comments - 14px, Silka

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/0b50477e-d9d7-447a-9079-017fdfc4c497)

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/1dfb20a3-21a4-49b8-9b24-2ec5a5b1443c)


![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/8631139d-d851-4db8-a990-11cc645df374)